CollapsibleExample/ExampleSubclassFilterViewController.m

```diff
@@ -36,20 +37,22 @@ - (void)createManagersAndPopulateData{
     
     MHCollapsibleViewManager *labels = [[MHCollapsibleViewManager alloc] initManagerWithAnimation:UITableViewRowAnimationMiddle topHierarchyTitle:@"Labels" tableView:self.tableView];
     
+    [labels setFilterArraysWithFirstArrayAsHeaderTitles:@[@"Labels"], self.returnLabelArray, nil];
     //sends double array for filternames and single array for header lines
-    [labels setDataWithFilterNames:self.returnLabelArray headerTitles:@[@"Labels"]];
+    //[labels setDataWithFilterNames:self.returnLabelArray headerTitles:@[@"Labels"]];
     
     MHCollapsibleViewManager *surveys = [[MHCollapsibleViewManager alloc] initManagerWithAnimation:UITableViewRowAnimationMiddle topHierarchyTitle:@"Surveys" tableView:self.tableView];
     
     NSArray *surveyQuestions = @[self.returnGuestbookArray, self.returnEngelsScaleArray, self.returnInternationalStudentsArray];
     NSArray *surveyList = @[@"Bridges@UCF Guestbook", @"Engels Scale", @"International Students"];
-    [surveys setDataWithFilterNames:surveyQuestions headerTitles:surveyList];
+    [surveys setFilterArraysWithFirstArrayAsHeaderTitles:surveyList, self.returnGuestbookArray, self.returnEngelsScaleArray, self.returnInternationalStudentsArray, nil];
     surveyList = nil;
     surveyQuestions = nil;
     
     MHCollapsibleViewManager *interactions = [[MHCollapsibleViewManager alloc] initManagerWithAnimation:UITableViewRowAnimationMiddle topHierarchyTitle:@"Interactions" tableView:self.tableView];
     //Index 0 is title
-    [interactions setDataWithFilterNames:@[self.returnInteractionsArray] headerTitles:@[@"Interactions"]];
+    [interactions setFilterArraysWithFirstArrayAsHeaderTitles:@[@"Interactions"], self.returnInteractionsArray, nil];
```

Comment:
@LizzyRandall  The previous way the method work is actually the standard way of writing methods in objc. The nil terminated method is a more confusing way of doing things for an iOS developer. If the method is confusing to ryan or others the solution is either a different name or documentation. If you document the method in the .h file anyone will be able to "option click" any reference to the method and see your documentation in a pop up.

An alternate name could be
```
[interactions setFiltersWithFilterNames:@[self.returnInteractionsArray] filterTitles:@[@"Interactions"]];
```
Or you could create a filter object that would match the two. I would actually preference that.
```
MHFilter *interactionsFilter = [[MHFilter alloc] init];
interactionsFilter.title = @"Interactions";
interactionsFilter.names = self.returnInteractionsArray; //if you make the above changes this would be interactionsFilter.names = self.interactions;
[interactions setFilters:@[interactionsFilter]]; //if you made filters a property and override the setFilters method you could write this as interactions.filters = @[interactionsFilter];
```